pkg/local_object_storage/metabase/metadata.go

```diff
@@ -0,0 +1,237 @@
+package meta
+
+import (
+	"bytes"
+	"encoding/base64"
+	"encoding/hex"
+	"errors"
+	"fmt"
+	"math/big"
+
+	objectcore "github.com/nspcc-dev/neofs-node/pkg/core/object"
+	cid "github.com/nspcc-dev/neofs-sdk-go/container/id"
+	"github.com/nspcc-dev/neofs-sdk-go/object"
+	oid "github.com/nspcc-dev/neofs-sdk-go/object/id"
+	"github.com/nspcc-dev/neofs-sdk-go/user"
+	"github.com/nspcc-dev/neofs-sdk-go/version"
+	"go.etcd.io/bbolt"
+)
+
+const (
+	metaPrefixA  = byte(iota)
+	metaPrefixBI // integer attributes
+	metaPrefixBS // all other attributes
+	metaPrefixC
```

Comment:
These better be named semantically.

Reply:
for sure, will do this finally. But for, now I'm used to naming from the task

Reply:
done